CHANGELOG.txt

```diff
@@ -183,6 +183,8 @@
                     - Add allowEventOverlaps property (boolean or method).
                     - Add allowEventCreate property (boolean).
         # Javascript:
+            * Use Object.assign instead of $.extend to remove the dependency to jQuery for : RGBColor, DateFaker, BrowserVersion & Assert.
+            * Use Object.assign instead of $.extend for sketch components.
```

Comment:
I don't think these items have to be in the changelog (it does not mean it's not interesting, but the changelog is  big even if we omit information which are not really useful).